neptun/src/sleepyinstant/mod.rs

```diff
@@ -50,9 +50,7 @@ impl Instant {
     /// Returns the amount of time elapsed from another instant to this one,
     /// or zero duration if that instant is later than this one.
     ///
-    /// # Panics
-    ///
-    /// panics when `earlier` was later than `self`.
+    /// Returns `Duration::ZERO` when `earlier` was later than `self`.
```

Comment:
This seems redundant now looking at line 51. 😅

Reply:
Yeah, I really should have taken a holistic look at this comment rather than just deleting the bad bit. I think we can drop everything after the last line.